lua/lualine/components/signify.lua

```diff
@@ -1,9 +1,7 @@
 local function signify()
    if vim.fn.exists('*sy#repo#get_stats') == 0 then return '' end
    local added, modified, removed = unpack(vim.fn['sy#repo#get_stats']())
-   if added == -1 then added = 0 end
-   if modified == -1 then modified = 0 end
-   if removed == -1 then removed = 0 end
+   if added == -1 then return '' end
```

Comment:
This indeed fixes the issue but could you explain why?

Reply:
This it the get_stats function
```vim
function! sy#repo#get_stats(...) abort 
  let sy = getbufvar(a:0 ? a:1 : bufnr(''), 'sy')
  return empty(sy) ? [-1, -1, -1] : sy.stats
endfunction
```